packages/canary-web-components/src/components/ic-data-table/ic-data-table.stories.mdx

```diff
@@ -25,6 +25,7 @@ import {
   Empty,
   EmptySlotted,
   Loading,
+  LoadingWithOverlay,
```

Comment:
This is imported but not used in the file - think there should be a story added to demonstrate the overlay?

Reply:
I was initially going to add a story but noticed there are no other stories specifically adjusting the loadingOptions. However, I have added `overlay` to the loading summary and it is possible to make the adjustment within the React storybook playground.